README.md

```diff
@@ -119,78 +118,56 @@ cargo build -p millau-bridge-node
 cargo build -p substrate-relay
 ```
 
-### Running
+### Running a Dev network
+
+We will launch a dev network to demonstrate relay from millau to rialto here
+
+#### Running from local scripts
 
 To run a simple dev network you'll can use the scripts located in
-[the `deployments/local-scripts` folder](./deployments/local-scripts). Since the relayer connects to
-both Substrate chains it must be run last.
+[the `deployments/local-scripts` folder](./deployments/local-scripts). Since the relayer connects to both Substrate chains it must be run last.
 
 ```bash
-# In `parity-bridges-common` folder
+
+# run nodes In **parity-bridges-common** folder
 ./deployments/local-scripts/run-rialto-node.sh
 ./deployments/local-scripts/run-millau-node.sh
-./deployments/local-scripts/relay-millau-to-rialto.sh
-```
-
-At this point you should see the relayer submitting headers from the Millau Substrate chain to the
-Rialto Substrate chain.
 
-### Local Docker Setup
-
-To get up and running quickly you can use published Docker images for the bridge nodes and relayer.
-The images are published on [Docker Hub](https://hub.docker.com/u/paritytech).
-
-To run the dev network we first run the two bridge nodes:
+# init bridge & relay headers from millau to rialto
+./deployments/local-scripts/relay-millau-to-rialto.sh
 
-```bash
-docker run -p 30333:30333 -p 9933:9933 -p 9944:9944 \
-           -it paritytech/rialto-bridge-node --dev --tmp \
-           --rpc-cors=all --unsafe-rpc-external --unsafe-ws-external
+# relay messages from millau to rialto
+./deployments/local-scripts/relay-messages-millau-to-rialto.sh
 
-docker run -p 30334:30333 -p 9934:9933 -p 9945:9944 \
-           -it paritytech/millau-bridge-node --dev --tmp \
-           --rpc-cors=all --unsafe-rpc-external --unsafe-ws-external
+# relay messages from rialto to millau for acknowledgement to work
```

Comment:
Don't think this is required, it's really  a detail that someone getting up and running doesn't need to worry about